src/benchmarks/bench_method/CMakeLists.txt

```diff
@@ -54,12 +54,11 @@ if(MSVC)
     target_compile_options(bench_method PRIVATE /Zm200)
 endif()
 
+set_compiler_warnings(bench_method "benchmarks")
```

Comment:
I am not 100% sure, but I think adding the flags, before setting the CXX standard, will lead to effect that in verbose builds, we will see, which c++ standard is used, as last argument to the compiler. I prefer seeing it as first.